dev-packages/browser-integration-tests/suites/integrations/featureFlags/launchdarkly/basic/test.ts

```diff
@@ -0,0 +1,48 @@
+import { expect } from '@playwright/test';
+
+import { sentryTest } from '../../../../../utils/fixtures';
+
+import { envelopeRequestParser, shouldSkipLaunchDarklyTest, waitForErrorRequest } from '../../../../../utils/helpers';
+
+const FLAG_BUFFER_SIZE = 100; // Corresponds to constant in featureFlags.ts, in browser utils.
+
+sentryTest('Basic test with eviction, update, and no async tasks', async ({ getLocalTestPath, page }) => {
+  if (shouldSkipLaunchDarklyTest()) {
+    sentryTest.skip();
+  }
+
+  await page.route('https://dsn.ingest.sentry.io/**/*', route => {
+    return route.fulfill({
+      status: 200,
+      contentType: 'application/json',
+      body: JSON.stringify({ id: 'test-id' }),
+    });
+  });
+
+  const url = await getLocalTestPath({ testDir: __dirname, skipDsnRouteHandler: true });
+  await page.goto(url);
+
+  await page.waitForFunction(bufferSize => {
+    const ldClient = (window as any).initializeLD();
+    for (let i = 1; i <= bufferSize; i++) {
+      ldClient.variation(`feat${i}`, false);
+    }
+    ldClient.variation(`feat${bufferSize + 1}`, true); // eviction
+    ldClient.variation('feat3', true); // update
+    return true;
+  }, FLAG_BUFFER_SIZE);
```

Comment:
This can just be `page.evaluate` right since we're not waiting for anything async inside